_dev/apps/ui/src/views/configuration-page.vue

```diff
@@ -29,5 +34,12 @@ export default defineComponent({
       return this.$store.getters['accounts/GET_PS_ACCOUNTS_CONTEXT_SHOPS'];
     },
   },
+  mounted() {
+    if (this.$refs.PreMonetizationPopin.$refs.modal) {
```

Comment:
why do you don't use your store or local storage her ? And I think you can only use internal .vue logic of each element and not using it in parent.